src/hub.html

```diff
@@ -32,6 +32,7 @@
         vr-mode-ui="enabled: false"
         stats-plus="false"
         action-to-event__mute="path: /actions/muteMic; event: action_mute;"
+        action-to-event__focus_chat="path: /actions/focusChat; event: action_focus_chat;"
```

Comment:
Nice, `action-to-event` was mostly meant as a way to port old code/components, but I guess this is maybe the preferred way to interact with anything without an existing tick function.